docs/how-to/default-profile.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Default profile
+legacyUrl: https://docs.ipfs.io/guides/examples/default-profile/
+description: Your profile defines which file-system and data-store your IPFS node will use, along with other configuration options. Learn how to set, change, and reset your profile.
+---
+
+# Configure a default profile and switching between profiles
+
+The profile of your IPFS node allows you to specify which file-system or data-store you want to use. Changing these options will affect the performance of your node. For example, if you want to have a faster datastore for your node, you can use the `badgerds` profile. If you're running your node on a low-power device like a Raspberry Pi, you can use the `low-power` profile. Profiles have been developed to customize your IPFS node to perform best under given conditions.
+
+## Find your current profile
+
+Your IPFS profile is found within your node's `config` file. The default location for the `config` file is `~/.ipfs/config`. If you have set an `$IPFS_PATH` variable, you can find your `config` file at `$IPFS_PATH/config`. Use `grep` to find the currently set profile:
+
+```bash
+cat ~/.ipfs/config | grep "prefix"
+# or
+cat $IPFS_PATH/config | grep "prefix"
+
+> "prefix": "flatfs.datastore",
+> "prefix": "leveldb.datastore",
+```
+
+IPFS uses the `flatfs` profile by default. `flatfs` is implemented using LevelDB, which is why you also see another line in the output with `leveldb.datastore`, but they both refer to `flatfs` datastore.
+
+If you previously configured your IPFS node to use another profile, let's say `badgerds`, the above command would output the following:
+
+```bash
```

Comment:
@alexxnica all of the docs in this project use bash for shell highlighting for linux/mac. It's also used by a much larger audience. 

Additionally, bash here doesn't refer to compatibility with bash but only syntax highlighting.

Reply:
@jay-dee7, I checked, and you're right; though not all files here use this approach, it's the most used one, just if you're curious:

` ```bash` is used 155 times, across 30 different files.
` ```sh` is used 7 times, across 4 different files.

I think we can discuss this further in the future, but let's leave it at that for now. I appreciate your prompt response though.

Reply:
In all honestly, we could probably use something like `json` here, since it formats things a bit nicer. But like you said @AlexxNica, this is a discussion for another time.